mappings/net/minecraft/world/gen/heightprovider/ConstantHeightProvider.mapping

```diff
@@ -1,5 +1,6 @@
 CLASS net/minecraft/unmapped/C_nyjqyqyk net/minecraft/world/gen/heightprovider/ConstantHeightProvider
 	FIELD f_lhvjjjgp ZERO Lnet/minecraft/unmapped/C_nyjqyqyk;
+	FIELD f_luladjgi CONSTANT_CODEC Lcom/mojang/serialization/Codec;
```

Comment:
Why isn't this just `CODEC` like in moj? (There are a few others of these, but I figured I shouldn't leave a comment on all of them)

Reply:
This was how yarn was. The mappings were originally lost when migrating hashed versions. This pr did the migration a second time, and the fields were saved